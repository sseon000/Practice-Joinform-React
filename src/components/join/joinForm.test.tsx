
import JoinForm from "./JoinForm.container"
import { render, fireEvent, screen } from "@testing-library/react"
import "@testing-library/jest-dom"

it("JoinForm이 잘 렌더링 됐는지 확인한다.", () => {
    const view = render(<JoinForm />);

    expect(view.container).toMatchSnapshot();
})

it("타이틀 회원가입 확인", () => {
    render(<JoinForm />);

    const title = screen.getByText("회원가입");
    expect(title).toBeInTheDocument();
})

/*
it("...check", () => {
    
})
*/

