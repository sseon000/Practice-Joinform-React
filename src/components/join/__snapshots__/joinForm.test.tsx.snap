// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JoinForm이 잘 렌더링 됐는지 확인한다. 1`] = `
<div>
  <div
    class="css-18irzgr"
  >
    <h3
      class="css-1wp369k"
    >
      회원가입
    </h3>
    <form>
      <input
        class="css-jx9hwa"
        id="id"
        placeholder="이메일을 입력해주세요."
        type="text"
      />
      <input
        class="css-jx9hwa"
        id="password"
        placeholder="비밀번호를 입력해주세요."
        type="password"
      />
      <input
        class="css-jx9hwa"
        id="passwordCheck"
        placeholder="비밀번호를 다시 입력해주세요."
        type="password"
      />
      <input
        class="css-mcgovs"
        id="submit"
        type="submit"
        value="가입하기"
      />
    </form>
  </div>
</div>
`;
